docs/monitors/_monitor-config.md

```diff
@@ -0,0 +1,174 @@
+
+<!--- Generated by to-integrations-repo script in Smart Agent repo, DO NOT MODIFY HERE --->
+<!--- GENERATED BY gomplate from scripts/docs/templates/monitor-main.md.tmpl --->
+
+# Monitor Configuration
+
+Monitors gather metrics from the host and from running applications.  They are
+configured in a list called `monitors` in the [main agent config
+file](../config-schema.md).  For example:
+
+```yaml
+monitors:
+ - type: elasticsearch
+   discoveryRule: 'port == 9200 && container_image =~ "elasticsearch"'
+ - type: cpu
+ - type: docker-container-stats
+   dockerURL: tcp://localhost:2379
+```
+
+[See here for a list of available monitors](#monitor-list)
+
+## Common Configuration
+
+The following config options are common to all monitors:
+
+| Config option | Default | Required | Type | Description |
+| --- | --- | --- | --- | --- |
+| `type` |  | no | `string` | The type of the monitor |
+| `discoveryRule` |  | no | `string` | The rule used to match up this configuration with a discovered endpoint. If blank, the configuration will be run immediately when the agent is started.  If multiple endpoints match this rule, multiple instances of the monitor type will be created with the same configuration (except different host/port). |
```

Comment:
Change instances of future tense to present tense, where applicable.

Reply:
See my previous comments.